warehouse/upload.go

```diff
@@ -1776,29 +1775,28 @@ func (job *UploadJobT) createLoadFiles(generateAll bool) (startLoadFileID, endLo
 				DestinationRevisionID:        job.warehouse.Destination.RevisionID,
 				StagingDestinationRevisionID: stagingFile.DestinationRevisionID,
 			}
-
 			if revisionConfig, ok := destinationRevisionIDMap[stagingFile.DestinationRevisionID]; ok {
 				payload.StagingDestinationConfig = revisionConfig.Config
 			}
-
 			if misc.ContainsString(warehouseutils.TimeWindowDestinations, job.warehouse.Type) {
 				payload.LoadFilePrefix = warehouseutils.GetLoadFilePrefix(stagingFile.TimeWindow, job.warehouse)
 			}
 
-			// set merged schema as upload schema if the load file type is parquet
-			if job.upload.LoadFileType == warehouseutils.LOAD_FILE_TYPE_PARQUET {
-				payload.UploadSchema = job.upload.MergedSchema
-			}
-
 			payloadJSON, err := json.Marshal(payload)
 			if err != nil {
 				panic(err)
 			}
```

Comment:
Outside the purview of the PR, but I saw an unnecessary panic here, should we handle it gracefully ?

Reply:
Ideally yes. But we need some more changes as well. So, didn't touch it.